client/src/components/App/Home/MakeBooking.tsx

```diff
@@ -473,11 +473,13 @@ const MakeBooking: React.FC = () => {
                           </div>
 
                           {day.userCanBook && (
-                            <BookButton
-                              onClick={(e) => handleCreateBooking(day.date, e.withParking)}
-                              availableCarPark={day.availableCarPark}
-                              buttonsLoading={buttonsLoading}
-                            />
+                            <div className="book">
+                              <BookButton
+                                onClick={(e) => handleCreateBooking(day.date, e.withParking)}
+                                availableCarPark={day.availableCarPark}
+                                buttonsLoading={buttonsLoading}
+                              />
+                            </div>
```

Comment:
Is this just always showing the button with dropdown now - and just greying out the parking option in the dropdown if not available?

Reply:
Incoming...

![Screenshot 2020-07-30 at 11 41 04](https://user-images.githubusercontent.com/10833481/88913840-9ad82400-d259-11ea-873e-c653e86f958d.png)